test_scripts/Polices/appID_Management/TS01.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary: 
+--     [OnAppInterfaceUnregistered] "APP_UNAUTHORIZED" in case of failed nickname validation after updated policies
+-- Description: 
+--     In case the application with "appName" is successfully registered to SDL
+--     and the updated policies do not have this "appName" in this app's specific policies
+--     (= "nicknames" filed in "<appID>" section of "app_policies")
+--     SDL must send OnAppInterfaceUnregistered (APP_UNAUTHORIZED) to such application.
+-- 
+-- Test:
+--     1. Preconditions:
+--        - Update Policy Table by assigning for "0000001" application additional nickname = "App1"
+--        - Create new mobile session and register "app_new" with nickname = "App1"
+--        - Default policies will be used for "app_new"
+--     2. Steps:
+--        - Update Policy Table by adding "New Application" nickname for "app_new"
+--     3. Expected result:
+--        - Response OnAppInterfaceUnregistered has the following data: success = true, reason = "APP_UNAUTHORIZED"
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+  config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]  
+  local Common = require("user_modules/shared_testcases/Policy_app_ID_Management_Common")
```

Comment:
Please rename the file Policy_app_ID_Management_Common as common name in user_modules/shared_testcases, for example:
testCasesForPolicyAppIdManagament